bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/handler/NanoleafControllerHandler.java

```diff
@@ -705,6 +709,17 @@ private void updateProperties() {
         }
     }
 
+    private void updateLayout(PanelLayout panelLayout) {
+        try {
```

Comment:
I would suggest to return fast from this method if no items are linked to the channel. This would reduce the runtime cost.

Reply:
I was thinking about this tonight, and updateState() all over OpenHab should maybe oinly take a java.util.function.Function which is only called if a channel was linked to something.